src/Sylius/Bundle/CoreBundle/spec/Templating/Helper/MoneyHelperSpec.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace spec\Sylius\Bundle\CoreBundle\Templating\Helper;
+
+use PhpSpec\ObjectBehavior;
+use Sylius\Bundle\CurrencyBundle\Templating\Helper\MoneyHelper;
+use Sylius\Bundle\MoneyBundle\Formatter\AmountFormatterInterface;
+use Sylius\Bundle\MoneyBundle\Templating\Helper\MoneyHelperInterface;
+use Sylius\Component\Currency\Context\CurrencyContextInterface;
+use Sylius\Component\Locale\Context\LocaleContextInterface;
+
+/**
+ * @mixin MoneyHelper
+ *
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ */
+class MoneyHelperSpec extends ObjectBehavior
+{
+    function let(
+        LocaleContextInterface $localeContext,
+        CurrencyContextInterface $currencyContext, 
+        AmountFormatterInterface $amountFormatter
+    ) {
+        $this->beConstructedWith($localeContext, $currencyContext, $amountFormatter);
+    }
+
+    function it_is_initializable()
+    {
+        $this->shouldHaveType('Sylius\Bundle\CoreBundle\Templating\Helper\MoneyHelper');
+    }
+
+    function it_implements_money_helper_interface()
+    {
+        $this->shouldImplement(MoneyHelperInterface::class);
+    }
+
+    function it_allows_to_format_money_in_different_currencies(
```

Comment:
Shouldn't it be divided into more sections:
 - [x] it formats money using context currency and locale if only amount is given
 - [x] it formats money using given currency and locale if they are given
 - [x] it formats money using default currency if not given
 - [x] it formats money using default locale if not given